ctms/sync.py

```diff
@@ -53,8 +55,17 @@ def sync_contact_with_acoustic(self, contact: ContactSchema):
 
     def _sync_pending_record(self, db, pending_record: PendingAcousticRecord):
         try:
-            contact: ContactSchema = get_acoustic_record_as_contact(db, pending_record)
-            is_success = self.sync_contact_with_acoustic(contact)
+            if self.is_acoustic_enabled:
+                contact: ContactSchema = get_acoustic_record_as_contact(
+                    db, pending_record
+                )
+                is_success = self.sync_contact_with_acoustic(contact)
+            else:
+                self.logger.debug(
+                    "Acoustic is not currently enabled. Records will be classified as successful and "
+                    "dropped from queue at this time."
+                )
+                is_success = True
 
```

Comment:
_acoustic FF check here_

If enabled continue w/ integration path; else, skip pulling contact and syncing, send success/True signal to delete records.